azure-resource-optimization-toolkit/nestedtemplates/Automation.json

```diff
@@ -0,0 +1,929 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "AutomationAccountName": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Assign a name for the Automation account"
+      }
+    },
+    "AutomationRegion": {
+      "type": "string",
+      "defaultValue": "West Europe",
+      "allowedValues": [
```

Comment:
I'm not seeing the changes in the param section (camelCase and allowedValues) - maybe they were undone via the template I sent...